### DISCUSSION TITLE
Goal 11: Make cities and human settlements inclusive, safe, resilient, and sustainable

### DISCUSSION THREAD

**Discussion by HSCunha** [Project-Resilience/platform#33] [General]
Chat about everything related to Goal 11: [Make cities and human settlements inclusive, safe, resilient, and sustainable](https://github.com/Project-Resilience/platform/blob/main/goals/goal_11.md)

**Comment by ofrancon** [comment]
One idea we discussed is how intersections can be made safer for pedestrians. Cities are working on different proposals:
- Berkeley, CA proposed [banning right turns on red](https://www.sfgate.com/bayarea/article/right-turn-on-red-ban-berkeley-update-18525300.php)
- San Francisco, CA [eliminated parking spaces to improve visibility at intersections](https://abc7news.com/post/daylighting-law-san-francisco-eliminating-14000-parking-spaces-cas-new-rule-takes-effect-heres-what-means/15538700/)
- [Crosswalk visibility enhancements](https://highways.dot.gov/safety/proven-safety-countermeasures/crosswalk-visibility-enhancements#:~:text=Improved%20Lighting,silhouette%20effect%20of%20the%20pedestrian.)
- etc.

and there's probably data:
- [The Crash Modification Factors Clearinghouse](https://cmfclearinghouse.fhwa.dot.gov/index.php)
- U.S. Department of Transportation [Secure Data Commons](https://www.transportation.gov/data/secure)
- [Using data to prevent crashes](https://www.waze.com/en/wazeforcities/casestudies/using-data-to-prevent-crashes/)

It should possible to learn what works / what doesn't work from the data, and help cities decide what they should implement.
Objectives can be:
- minimize number of accidents
- minimize impact on traffic (speed, congestion, etc.)

There are differences between cities and countries, but it's probably possible to generalize to some extend. And help cities avoid mistakes other cities have made, and try policies that worked.

**Reply by Jarrattp31** [reply]
There is some great information here about how roundabouts are safer then traffic lights in the US by WEF: 
https://www.weforum.org/stories/2021/12/roundabouts-save-more-lives-than-traffic-lights/ and data here
https://roundabouts.kittelson.com/Home/Reports

**Comment by KanthiNimm** [comment]
_Resources to access information on pedestrian safety and policies implemented by different countries._

Global status report on road safety 2023 - _https://www.who.int/publications/i/item/9789240086517_
Few sites that provide data on global pedestrian safety - irap.org, grsproadsafety.org, UNECE Transport Statistics Database, European Commission CARE Database, itf-oecd.org

**Comment by danyoungday** [comment]
Operation Canopy Plan:
Lyon is planting trees to raise the % area covered in order to reduce the temperature in the city.
This could be optimized if we can find data on where trees are planted as well as temperature readings over time.
[https://www.paysalia.com/en/blog/green-city/lyon-canopy-plan-urban-forest](https://www.paysalia.com/en/blog/green-city/lyon-canopy-plan-urban-forest)

**Reply by GPaolo** [reply]
lots of resources here https://land.unece.org/treesincities/knowledge-hub

**Comment by GPaolo** [comment]
Just found this digital twin resource for cities https://city.gate-ai.eu/

**Reply by GPaolo** [reply]
There is also this company run by Stanford researchers that built VillageOS, a digital twin for a fully sustainable village https://www.regenvillages.com/

**Reply by GPaolo** [reply]
These guys did a map/simulator of Paris https://mira.world/about/

**Comment by deepsaia** [comment]
There's WRI - data for cool cities: https://www.wri.org/initiatives/data-cool-cities

**Comment by ofrancon** [comment]
@GPaolo  @deepsaia  @danyoungday  Lots of info in this study:
[The 2026 Europe Cities Report of the Lancet Countdown on health and climate change: cities at the forefront of climate change action](https://www.thelancet.com/journals/lanpub/article/PIIS2468-2667(26)00100-3/fulltext)
